ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -122,7 +126,7 @@ public void write(OutputStream out) {
   }
 
   private void write(OutputStream output, GenericRow row) throws IOException {
-    objectMapper.writeValue(output, StreamedRow.row(row));
+    output.write(jsonSerializer.serialize("", row));
```

Comment:
As above, I think _not_ outputting a StreamedRow here is a breaking change. Why not just send a StreamedRow? This gives us more freedom in the future to change StreamedRow as we need without having to change GenericRow.

The `outputException()` method continues to output a StreamedRow.  We should stay consistent and _always_ send back a StreamedRow. That way, deserializing on the other end is trivial and doesn't require any hand crafted 'if (this) then that, else this" style logic.

Reply:
Reverted this to use StreamedRow always as discussed.